Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -2356,16 +2373,18 @@ void RenderFlexibleBox::alignFlexLines(FlexLineStates& lineStates, LayoutUnit ga
 
     ContentPosition position = style().resolvedAlignContentPosition(contentAlignmentNormalBehavior());
     ContentDistribution distribution = style().resolvedAlignContentDistribution(contentAlignmentNormalBehavior());
+    OverflowAlignment safety = style().alignContent().overflow();
 
-    if (position == ContentPosition::FlexStart && !gapBetweenLines)
+    if (position == ContentPosition::FlexStart && !gapBetweenLines
+        && !(safety == OverflowAlignment::Safe))
```

Comment:
Yeah, I'm not sure why I didn't do that. :) Note that "is safe" is not the opposite of "is unsafe" because there's a "is default behavior" option as well.